### PR TITLE
enable meta.delegates to enforce KEYWORD_ONLY for kwargs from

### PR DESCRIPTION
**Pull request by EmbraceLife**
The details of this PR can be found in this forum [thread](https://forums.fast.ai/t/python-help/97525/22?u=daniel).

### CODE REVIEW

File: fastcore/meta.py [hunk 1]
```diff
@@ -120,20 +120,21 @@ def _f(f):
         k = sigd.pop('kwargs')
         s2 = {k:v for k,v in inspect.signature(to_f).parameters.items()
```
Comment (jph00) [inline fastcore/meta.py:121]:
```suggestion
        s2 = {k:v.replace(kind=inspect.Parameter.KEYWORD_ONLY) for k,v in inspect.signature(to_f).parameters.items()
```

If you made this change, would that mean you could then remove the line you added?

**Review by jph00 — CHANGES_REQUESTED** [review]
Many thanks for the PR. I've made one suggested change (but haven't tested it -- so take a look and see if it works for you!).

You've accidentally added the whole _docs directory to your PR. Could you please remove it? I've just updated .gitignore so that this shouldn't happen again in the future.

File: fastcore/meta.py (hunk 1 above)
Comment (EmbraceLife) [reply fastcore/meta.py]: Yes, this change works and my added line can be removed. Thanks Jeremy!

Comment (jph00) [reply fastcore/meta.py]: Great. Just reply here when you've removed the `_docs` folder and we should be ready to go!

**Comment by EmbraceLife** [comment]
I have reverted my first troublesome commit and then only run `nbdev_prepare` this time without running all the cells, nor running `nbdev_docs` and `nbdev_preview`, so that I didn't generate so many differences.

**Comment by jph00** [comment]
Good job!

**Comment by EmbraceLife** [comment]
<p>Thank you Jeremy! This is my first contribution to fastcore, Wow!</p>

**Pull request merged**